xbmc/cores/VideoPlayer/DVDCodecs/Audio/DVDAudioCodecPassthrough.cpp

```diff
@@ -145,7 +145,8 @@ bool CDVDAudioCodecPassthrough::AddData(const DemuxPacket &packet)
 
     if (used != iSize)
     {
-      if (m_backlogBufferSize < (iSize - used))
+      const int t_diff = iSize - used;
```

Comment:
if you allow this term to be < 0, you have to handle this case. but it would be a waste of resources because it won't be < 0

Reply:
Yes. I figured. But it was the only way to avoid the warning. I didn't really want to change this but according the the jenkins build I had 1 warning too many on several platforms and this was literally the only warning I could address. I can just drop the commit (this change is in a separate commit). Just let me know if that's what you want.